packages/truffle-box/lib/utils/unbox.js

```diff
@@ -1,90 +1,73 @@
-const fs = require("fs-extra");
+const fse = require("fs-extra");
 const path = require("path");
 const ghdownload = require("github-download");
-const request = require("request");
+const rp = require("request-promise-native");
 const vcsurl = require("vcsurl");
 const parseURL = require("url").parse;
-const exec = require("child_process").exec;
+const execSync = require("child_process").execSync;
```

Comment:
Not really a big deal or something that needs to hold this back, but how do you like the destructured version of this?
`const { execSync } = require("child_process");`

Reply:
Good idea, I think it's cleaner.